### PR TITLE
fix(Button): adjust loading state to initial button size

### PR DESCRIPTION
**Pull request by talkor**
Screenshot 2023-10-25 at 20 00 40

https://monday.monday.com/boards/3532714909/pulses/5350721960

### CODE REVIEW

**Comment by shlomitc** [comment]
Should the button have a min-width? In very short text it makes sense the button will look a bit larger
image

**Comment by talkor** [comment]
> Should the button have a min-width? In very short text it makes sense the button will look a bit larger image

The min-width hasn't changed in this PR but I will verify with Meirav

**Comment by shlomitc** [comment]
also - what should happen if the success text is smaller than the children text?
From my test - it shrinks to the success text, but I think we said it should stay the same size

**Comment by talkor** [comment]
> also - what should happen if the success text is smaller than the children text? From my test - it shrinks to the success text, but I think we said it should stay the same size

We have a story for this, the text becomes shorter and the button remains the same size, this also hasn't changed
Screenshot 2023-10-26 at 9 21 42

File: src/components/Button/Button.module.scss [hunk 1]
```diff
@@ -26,6 +26,21 @@ $disabled-on-primary-color-opacity: 0.5;
   /* Will be updated dynamically after render by js */
   --element-width: 32;
   --element-height: 32;
+  min-width: var(--element-width);
```
Comment (talkor) [inline src/components/Button/Button.module.scss:29]: Moved from line 50

Comment (shlomitc) [reply src/components/Button/Button.module.scss:29]: I don't see the effect of this. are you sure it's working? it's missing the `px`

Comment (shlomitc) [reply src/components/Button/Button.module.scss:29]: in addition, the `element-width` will overridden after being calculated in the js

Comment (talkor) [reply src/components/Button/Button.module.scss:29]: Oh yeah your'e right, the definition is missing the `px`, it is being overridden with JS (`Button.tsx:162-163`) so it's used as a default/fallback value, I will add `px` to the definition of the variable

Comment (talkor) [reply src/components/Button/Button.module.scss:29]: It did not work before as the fallback value of 32px as the value was illegal, now when I changed it to be a valid value if messes up some component that use Button with a smaller width than 32px, so I think we should remove the default value at all

**Review by shlomitc — APPROVED** [review]

**Pull request merged**